src/main/resources/org/csanchez/jenkins/plugins/kubernetes/pipeline/scripts/kill-process-by-id.ps1

```diff
@@ -0,0 +1,68 @@
+param(
+    [string]$processId
+)
+
+Add-Type -TypeDefinition @"
+using System;
+using System.Runtime.InteropServices;
+using System.Diagnostics;
+
+public class ProcessControl {
+    [DllImport("kernel32.dll")]
+    public static extern bool AttachConsole(int dwProcessId);
+    [DllImport("kernel32.dll")]
+    public static extern bool SetConsoleCtrlHandler(ConsoleCtrlDelegate HandlerRoutine, bool Add);
+    public delegate bool ConsoleCtrlDelegate(int CtrlType);
+    [DllImport("kernel32.dll")]
+    public static extern bool GenerateConsoleCtrlEvent(uint dwCtrlEvent, uint dwProcessGroupId);
+    [DllImport("kernel32.dll")]
+    public static extern bool FreeConsole();
+}
+"@
```

Comment:
Could be moved to a separate `*.cs` file, though it is just signatures so I guess it does not much matter for legibility.